awx/ui/src/screens/NotificationTemplate/NotificationTemplateDetail/NotificationTemplateDetail.js

```diff
@@ -20,13 +20,20 @@ import {
 import CodeDetail from 'components/DetailList/CodeDetail';
 import DeleteButton from 'components/DeleteButton';
 import ErrorDetail from 'components/ErrorDetail';
-import { NotificationTemplatesAPI } from 'api';
+import { NotificationTemplatesAPI, NotificationsAPI } from 'api';
 import useRequest, { useDismissableError } from 'hooks/useRequest';
+import StatusLabel from 'components/StatusLabel';
 import hasCustomMessages from '../shared/hasCustomMessages';
 import { NOTIFICATION_TYPES } from '../constants';
 
+const NUM_RETRIES = 25;
+const RETRY_TIMEOUT = 5000;
+
 function NotificationTemplateDetail({ template, defaultMessages }) {
   const history = useHistory();
+  const [testStatus, setTestStatus] = useState(
+    template.summary_fields?.recent_notifications[0]?.status ?? undefined
```

Comment:
Do we want this to be `undefined` or `null`?

Reply:
the Status Label shows `undefined` if if can't match the status to something is knows.  Thats why I made it undefined here.